media/insert.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Security-Policy"
+        content="default-src 'none'; script-src vscode-resource:; style-src vscode-resource:;">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css"href="%STYLESRC%">
+    <title>PowerShell: Insert Command</title>
+</head>
+
+<body>
+    <form id="CommandParameters" onsubmit="mySubmit()">
+        <h1 id="CommandName"></h1>
+        <h2>Parameter Sets</h2>
+        <div id="ParameterSets"></div>
+        <input type="submit" />
+    </form>
+    <script src="%SCRIPTSRC%"></script>
+
+</body>
+
+</html>
```

Comment:
nit: always have a newline at EOF
```suggestion
</html>

```